iXBRLViewerPlugin/viewer/src/js/unit.js

```diff
@@ -13,13 +12,17 @@ export class Unit {
                 .split('/');
         this._numerators = split[0].split('*');
         this._denominators = split.length > 1 ? split[1].split('*') : [];
-        this._isMonetary = Boolean(this._numerators.find(n => this._report.qname(n).namespace === "http://www.xbrl.org/2003/iso4217"));
-        this._label = split.map(x => {
-                const part = x.split('*').map(y => {
-                    return titleCase(y.split(':')[1]);
-                }).join('*');
+        this._isSimple = this._numerators.length === 1 && this._denominators.length === 0;
+        this._isMonetary = Boolean(this._numerators.find(n => this._report.qname(n).namespace === NAMESPACE_ISO4217));
```

Comment:
Pre-existing, but I think a monetary unit should be a simple unit (single numerator, no denominator) with the measure being ISO 4217 namespace.